site/src/component/SearchFilters/SearchFilters.scss

```diff
@@ -0,0 +1,42 @@
+@use '../../globals.scss';
+
+.filter-group {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  padding-top: 8px;
+}
+
+p.filter-placeholder {
+  color: var(--text-secondary);
+  margin: 0;
+}
+
+.MuiFormControl-root {
+  height: fit-content;
+  width: fit-content;
+
+  input {
+    width: unset;
+  }
+}
+
+/* Hacky way to override default MUI styling; MUI applies defaults with high selector specificity */
+div.MuiInputBase-root.MuiOutlinedInput-root.MuiInputBase-colorPrimary.MuiInputBase-fullWidth.MuiInputBase-formControl.MuiInputBase-adornedEnd.MuiAutocomplete-inputRoot {
+  padding: 4px 40px 4px 10px;
+}
+
+input#multiple-limit-tags.MuiInputBase-input.MuiOutlinedInput-input.MuiInputBase-inputAdornedEnd.MuiAutocomplete-input.MuiAutocomplete-inputFocused {
+  font-size: 11px;
+  padding: 0;
+}
+
+.MuiInputBase-sizeSmall.MuiInputBase-adornedStart.MuiInputBase-adornedEnd.MuiAutocomplete-inputRoot.css-tzga38-MuiInputBase-root-MuiOutlinedInput-root {
+  padding-right: 6px;
+}
```

Comment:
I have a hard time telling what is actually being targeted here. if it's for one specific button, we should add a className to that button and instead target it inside `.search-filters .that-class {}`